kits/agp1/contracts/AGP1Kit.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity 0.4.24;
+
+import "@aragon/apps-finance/contracts/Finance.sol";
+import "@aragon/apps-vault/contracts/Vault.sol";
+import "@aragon/apps-voting/contracts/Voting.sol";
+import "@aragon/apps-shared-minime/contracts/MiniMeToken.sol";
+
+import "@aragon/os/contracts/apm/APMNamehash.sol";
+import "@aragon/os/contracts/common/IsContract.sol";
+
+import "@aragon/kits-base/contracts/KitBase.sol";
+
+
+contract AGP1Kit is KitBase, APMNamehash, IsContract {
+
+    uint64 constant public MAIN_VOTING_SUPPORT = 50 * 10**16; // > 50%
+    uint64 constant public MAIN_VOTING_QUORUM = 0; // Just 1 vote is enough
+    uint64 constant public MAIN_VOTING_VOTE_TIME = 48 hours;
+
+    uint64 constant public META_TRACK_VOTING_SUPPORT = 666666666666666667; // > two thirds
+    uint64 constant public META_TRACK_VOTING_QUORUM = 0; // Just 1 vote is enough
+    uint64 constant public META_TRACK_VOTING_VOTE_TIME = 48 hours;
+
+    uint64 constant public FINANCE_PERIOD_DURATION = 90 days;
```

Comment:
I think I would make it `365.25 days / 4` so in the long run there are 4 periods per year. Alternatively we can make it `365 days / 4` which would be less weird.